src/graph.js

```diff
@@ -209,8 +210,16 @@ export default class Graph {
     return items.reduce((sum, entry) => sum + parseFloat(entry.state), 0);
   }
 
+  _delta(items) {
+    return this._maximum(items) - this._minimum(items);
+  }
+
   _lastValue(items) {
-    return parseFloat(items[items.length - 1].state) || 0;
+    if (this._calcPoint(items) === this._delta(items)) {
```

Comment:
Can you please explain this condition? Why for delta the last value should always be 0?

I think it is better to store aggregate func name as property and do the check here which is more clear IMO. Now we have redundant computation which is not clear (adding a comment would be useful if eventually you'll leave it)

Reply:
Honestly i'm not very proud of that code. Your solution seems way better.

If the sensor value are not updated anymore (for instance the meter is down) there is no new value. In this cas with the initial code the selected value is the last value of the meter. With the delta function if there is no updated value it should show 0 (no change in value).